modules/meeting/lib/open_project/meeting/engine.rb

```diff
@@ -40,7 +40,10 @@ class Engine < ::Rails::Engine
       project_module :meetings do
         permission :view_meetings, meetings: %i[index show], meeting_agendas: %i[history show diff],
                                    meeting_minutes: %i[history show diff]
-        permission :create_meetings, { meetings: %i[new create copy] }, require: :member
+        permission :create_meetings,
+                   { meetings: %i[new participants_section create copy] },
```

Comment:
With the change to turbo-stream, there no longer is a `participants_section` method added to the controller so this addition can be reverted.